browser/src/canvas/sections/CommentSection.ts

```diff
@@ -461,6 +461,14 @@ export class Comment extends CanvasSectionObject {
 	}
 
 	private textAreaKeyDown (ev: any): void {
+		// Save and print should be gloablly available
+		if (ev && ev.ctrlKey && (ev.code === 'KeyS' || ev.code === 'KeyP')) {
```

Comment:
can we somehow move it to https://github.com/CollaboraOnline/online/blob/master/browser/src/global.d.ts#L245 ?